package.json

```diff
@@ -178,4 +178,4 @@
     "wait-on": "7.2.0"
   },
   "packageManager": "yarn@4.1.1"
-}
+}
```

Comment:
There should be a newline here, please configure your editor to add it, if not configured already.